pkg/k8s/constants.go

```diff
@@ -35,6 +35,8 @@ const (
 	TimeToLeaseRenewal                   = 20 * time.Minute
 	MeanReconcileReenqueuePeriod         = 10 * time.Minute
 	JitterFactor                         = 2.0
+	Paused                               = "Paused"
+	PausedMessage                        = "The resource is up to date but actuation onto the cloud provider is Paused"
```

Comment:
I personally think this paused message may cause some potential confusion ("actuation onto the cloud provider" may not be a very common concept) though I don't have a better suggestion. Though we can/should always explain it in our public doc.

@jingyih @gemmahou do you have any thoughts on it?